src/components/Activity/NewRequests.js

```diff
@@ -72,24 +83,34 @@ class NewRequests extends Component {
     );
   }
 
+  closeAllExceptCurrent(id) {
```

Comment:
Okay. First, This function will executed by child component with it's id.
Then change all of 'close' properties to true except current swipe button which is opened.
In other words, the value has same id as given parameter will have 'false' in close property.
Lastly, it updates data source by calling setState() to re-render our listview.